packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json

```diff
@@ -631,7 +631,8 @@
   "modals.sencbtc.maximumamountmessage": "Not enough funds. Use",
   "modals.sencbtc.maximumfeemessage": "Unnecessarily high fee. Use",
   "modals.sencbtc.minimumamountmessage": "Below minimum amount. Use",
-  "modals.sencbtc.minimumfeemessage": "Low custom fee not recommended. Use",
+  "modals.sencbtc.minimumfeemessage": "Low fee not recommended. Use",
+  "modals.sencbtc.minimumonesatoshimessage": "Invalid (< 1sat/b)",
```

Comment:
is "(< 1sat/b)" translatable? or should that just be in its own Text component, untranslated.?

Reply:
mmm yep you are right, we can't translate that, so it should go in as a dynamic value

Reply:
updated